testsuite/oauth/authorize/base.py

```diff
@@ -128,9 +158,10 @@ def open(self):
         self.browser = self._browser()
 
     def close(self):
-        """ Close the selenium webdriver.
+        """ Close the virtual display.
```

Comment:
Selenium webdriver is still being closed here. Maybe a more generic comment like "close the virtual browser"?